scripts/launch/demo.sh

```diff
@@ -170,7 +170,7 @@ while [[ $i -lt $panesCnt ]]; do
           conf_file=$WALLET_CONFIG
       fi
       wallet_args=" --tlscert $base/../tls-files/server.crt --tlskey $base/../tls-files/server.key --tlsca $base/../tls-files/ca.crt $wallet_flush" # --wallet-rebuild-db'
-      wallet_args="$WALLET_EXTRA_ARGS --new-wallet $wallet_args --wallet-address 127.0.0.1:8091"
+      wallet_args="$WALLET_EXTRA_ARGS --legacy-wallet $wallet_args --wallet-address 127.0.0.1:8091"
```

Comment:
@erikd Was the addition of this flag intentional? I'm thinking we might have better luck to simply omit it, as for `1.3.1` stuff it would default to the legacy layer anyway, but for the code in `develop` it should pick the new data layer, which seems like a sensible thing to be working with. Thoughts?

Reply:
So I can simply drop the `--legacy-wallet` and it will work correctly? Let me try that.

Reply:
If I drop that the node in the wallet pane dies with:
```
[node:INFO:ThreadId 4] [2018-09-12 22:35:38.10 UTC] Passive Wallet kernel initialized.
[node:DEBUG:ThreadId 33] [2018-09-12 22:35:38.10 UTC] Starting wallet worker.
[node:DEBUG:ThreadId 33] [2018-09-12 22:35:38.10 UTC] Stopping wallet worker.
[node:ERROR:ThreadId 4] [2018-09-12 22:35:38.11 UTC] logException: Network.Socket.bind: resource busy (Address already in use)
cardano-node: Network.Socket.bind: resource busy (Address already in use)
[launcher:WARNING] [2018-09-12 22:35:38.12 UTC] The node has exited with ExitFailure 1
[launcher:NOTICE] [2018-09-12 22:35:38.12 UTC] Finished serverScenario
```